### ISSUE TITLE
Text cut off by not word wrapping

### ISSUE THREAD

**Issue by Victor239** [nextcloud-libraries/nextcloud-vue#6177]
I was told the issue [reported here](https://github.com/nextcloud/tasks/issues/2705) is due to a component from this library.

**Comment by susnux** [comment]
cc @nextcloud-libraries/designers

**Comment by szaimen** [comment]
I agree it would be good to let them wrap.at least one additional line...
